### ISSUE TITLE
Use OnkoKB-annotator offline

### ISSUE THREAD

**Issue by williambrandler** [oncokb/oncokb-annotator#224]
Is there any way or workaround to use OnkoKB-annotator offline, i.e. in a air-gapped system without access to the public internet?

**Comment by calvinlu3** [comment]
Hi, the oncokb-annotator is a wrapper around our API. It needs to reach out to our API, so internet access is necessary.

**Comment by williambrandler** [comment]
yeah I figured, are there any alternatives that do not require internet access?

**Comment by calvinlu3** [comment]
We generally do not allow users to have a copy of our database in a local installation environment. The annotator must be used with our public api, so no alternatives in an "offline mode".
